market_maker_keeper/cex_api.py

```diff
@@ -49,8 +49,6 @@ def __init__(self, arguments: Namespace, pyex_api: PyexAPI):
 
         self.history = History()
 
-        self.init_order_book_manager(arguments, pyex_api)
```

Comment:
Start logic was moved to the subclass to ensure that the overriding order_book_manager would always be used.